iothub_service_client/src/iothub_messaging_ll.c

```diff
@@ -457,15 +457,16 @@ static char* createAuthCid(IOTHUB_MESSAGING_HANDLE messagingHandle)
     }
     else
     {
-        const char* AMQP_SEND_AUTHCID_FMT = "iothubowner@sas.root.%s";
-        size_t authCidLen = strlen(AMQP_SEND_AUTHCID_FMT) + strlen(messagingHandle->iothubName);
+        const char* AMQP_SEND_AUTHCID_FMT = "%s@sas.root.%s";
+        const int AMQP_SEND_AUTHCID_FMT_LENGTH = 10;
```

Comment:
>10 [](start = 49, length = 2)

Should this be changed to strlen(AMQP_SEND_AUTHCID_FMT)?

Reply:
better yet sizeof() :)

---
In reply to: [310311464](https://github.com/Azure/azure-iot-sdk-c/pull/1094#discussion_r310311464) [](ancestors = 310311464)